src/routes/api/api-router.ts

```diff
@@ -82,16 +82,23 @@ ApiRouter.post(
 		const offset = Number(req.body.offset) || 0;
 		// only resync installations whose "updatedAt" date is older than x seconds
 		const inactiveForSeconds = Number(req.body.inactiveForSeconds) || undefined;
+		// How far back to fetch commit history(main and branch) in milliseconds.
+		const commitTimeLimit = Number(req.body.commitTimeLimit) || undefined;
 
 		if (!statusTypes && !installationIds && !limit && !inactiveForSeconds){
 			res.status(400).send("please provide at least one of the filter parameters!");
 			return;
 		}
 
+		if (commitTimeLimit !== undefined && (commitTimeLimit <= 0 || isNaN(commitTimeLimit))){
```

Comment:
if you don't do `|| undefined` above, you can just do `if(!commitTimeLimit || commitTimeLimit <=0)`

Reply:
actually, you can do the same even with `|| undefined` above